### DISCUSSION TITLE
RDB$DB_KEY added to "Browse data"

### DISCUSSION THREAD

**Discussion by pgfiore** [mariuz/flamerobin#395] [Q&A] [answered]
Hi guys, sorry for bothering you with such a stupid question... When I "Browse data" (right-click from the tables tree), FR 0.9.11 adds the RDB$DB_KEY pseudo-column to the select statement. It depends on the tables; I mean, it always happens, but only on a few specific tables.
Need help understanding this behavior (if I admit it's really nothing). Thank you
pf

**Comment by arvanus** [comment, marked as answer]
Does this tables have defined primary key?

**Comment by pgfiore** [comment]
I've received the whole DB and have yet to investigate the details... And you're right, no PK at all. Thanks for this clue.
